tools/assetpipelineeditor/exportdialog.cpp

```diff
@@ -0,0 +1,88 @@
+#include "exportdialog.h"
+#include "ui_exportdialog.h"
+
+#include <private/gltf2exporter_p.h>
+
+#include <QFile>
+#include <QFileInfo>
+#include <QDir>
+#include <QJsonDocument>
+#include <QFileDialog>
+#include <QStyle>
+#include <QPushButton>
+#include <QMessageBox>
+
+ExportDialog::ExportDialog(
+        Kuesa::GLTF2Exporter &exporter,
```

Comment:
Parameters would be better on a single line.

Reply:
done